birdhouse/deployment/install-automated-deployment.sh

```diff
@@ -71,14 +71,18 @@ set -x
 sudo cp -v $REPO_ROOT/birdhouse/deployment/triggerdeploy.sh /usr/local/sbin/
 
 
-CRON_FILE="/etc/cron.d/PAVICS-deploy"
+CRON_FILE="/etc/cron.d/Birdhouse-deploy"
+
+. "${REPO_ROOT}/birdhouse/read-configs.include.sh"
+
+read_basic_configs_only
 
 export CRON_FREQUENCY_TXT="$CRON_FREQUENCY_TXT"
 export CRON_SCHEDULE="$CRON_SCHEDULE"
-export OWNER_PAVICS_CHECKOUT="$REPO_OWNER"
-export PATH_TO_PAVICS_CHECKOUT="$REPO_ROOT"
+export OWNER_BIRDHOUSE_CHECKOUT="$REPO_OWNER"
+export PATH_TO_BIRDHOUSE_CHECKOUT="$REPO_ROOT"
```

Comment:
I'm ok with this... @tlvu do you see any issues with this?

Reply:
I am okay with Francis normalized new names.  Since we are doing this gigantic rename effort, might as well enforce some naming policies and normalization.